spacy/ml/tb_framework.pyx

```diff
@@ -302,8 +304,8 @@ def _forward_fallback(
     backprop_feats,
     seen_mask,
     is_train: bool,
-    actions: Optional[List[Ints1d]]=None,
-    max_moves: int=0):
+    actions: Optional[List[Ints1d]] = None,
+        max_moves: int = 0):
```

Comment:
This indent seems strange?

Reply:
Right, probably a mistake in my linting fixes. See https://github.com/explosion/spaCy/pull/12842#discussion_r1269043556.